docs/content/en/cookbook-modules.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Modules structure
+description: 'Create first modules'
+position: 12
+category: Cookbook
+---
+
+For the project to work properly you need to add the first modules to the structure.
+
+## Create `modules` directory
+
+Create an `modules` directory to keep the modules in it.
```

Comment:
repeated again, consider of moving it to the moving it into "Basic of module" or sth like that, and then like for `@example` or `@example2` link rest.

Reply:
This whole file is an introduction to creating modules. Things to prepare modules are described here.